src/Hl7.Fhir.Core/Model/Generated/Template-ModelInfo.cs

```diff
@@ -632,6 +632,7 @@ public static string Version
                 { typeof(Hl7.Fhir.Model.ValueSet), "ValueSet" },
                 { typeof(Hl7.Fhir.Model.VerificationResult), "VerificationResult" },
                 { typeof(Hl7.Fhir.Model.VisionPrescription), "VisionPrescription" },
+				 { typeof(Hl7.Fhir.Model.FhirPatchParameters), "Parameters" },
```

Comment:
This is not allowed. This file `Template-ModelInfo` will be generated, so all changes here will be overriden by a new generate. Also `FhirPathParameters` is not a standard Fhir resource (https://www.hl7.org/fhir/resourcelist.html)

Reply:
This was regenerated. There is a corresponding change to the template: `src/Hl7.Fhir.Core/Model/Template-Model.tt`.
Though I am not sure how to achieve what you are suggesting.